src/queue.js

```diff
@@ -0,0 +1,44 @@
+export default class Queue {
```

Comment:
I'm wondering why the syntax here isn't the same with the test? One example is in line 2 "constructor() {" is different to call out the function from the test "() =>"  let me know if it's better to be consistent with this or if the test syntax is different. Thanks!